bin/deliver_reviewed_data.py

```diff
@@ -437,6 +439,9 @@ def deliver_data(self):
                 project_report = None
             if project_report and os.path.exists(project_report):
                 project_to_reports[project] = project_report
+                # Copy the project report to the log folder
+                project_report_dest = os.path.join(self.delivery_report_repository, os.path.basename(project_report))
```

Comment:
@tcezard May you please confirm whether this is where the following requirement is fulfilled please?

"Archive the project report to a repository folder after generation."